.github/workflows/reusable-uki-test.yaml

```diff
@@ -153,18 +153,23 @@ jobs:
           rm -rf /tmp/unpacked
           sudo luet util unpack ${{ env.IMAGE_NAME }} ./unpacked --local
           new_version="mynewversion"
+          echo "Current release info:"
```

Comment:
I'd leave these here, makes it easy to read what artifact teh build is working on in detail